rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -249,7 +249,7 @@ void logCommitFederation() {
         String oldFederationBtcAddress = oldFederation.getAddress().toBase58();
         byte[] newFederationFlatPubKeys = flatKeysAsByteArray(newFederation.getBtcPublicKeys());
         String newFederationBtcAddress = newFederation.getAddress().toBase58();
-        long newFedActivationBlockNumber = executionBlock.getNumber() + CONSTANTS.getFederationActivationAge();
+        long newFedActivationBlockNumber = executionBlock.getNumber() + CONSTANTS.getFederationActivationAge(activations);
```

Comment:
I'm not opposed to writing a new test, but it will be a kind of integration test(like the tests written for the powpeg). We already have specific unit tests for this method, testing exclusively it. See: https://github.com/rsksmart/rskj/blob/increase-new-fed-activation-age/rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java#L69-L105